repository/git.go

```diff
@@ -986,36 +986,11 @@ func (repo *GitRepo) FetchAndReturnNewReviewHashes(remote, notesRefPattern,
 	return updatedReviews, nil
 }
 
-func (repo *GitRepo) GetColorBool(name string) bool {
-	ok := repo.runGitCommandInline("config", "--get-colorbool", name)
-	return (ok == nil)
-}
-
-func (repo *GitRepo) GetColor(name, default_value string) string {
-	var res string
-	var ok error
-	if default_value == "" {
-		res, ok = repo.runGitCommand(
-			"config",
-			"--type=color",
-			"-z",
-			"--get",
-			name,
-		)
-	} else {
-		res, ok = repo.runGitCommand(
-			"config",
-			"--type=color",
-			"-z",
-			"--default",
-			default_value,
-			"--get",
-			name,
-		)
-	}
+func (repo *GitRepo) GetColorBool(name string) (bool, error) {
+	err := repo.runGitCommandInline("config", "--get-colorbool", name)
+	return (err == nil), nil
```

Comment:
I don't think it makes sense for this function to return (bool, error) as opposed to bool only, but I understand from your comment that this is a convention so I oblige.